### PR TITLE
API-2793 Update appeals api status swagger documentation

### PR DESCRIPTION
**Pull request by ElleeB**
## Description of change
After comparing our documentation with Caseflow's schema, I updated our yml file to reflect discrepancies and to add clarity. A few changes were made to the syntax, following Swagger standards accordingly. 

## Original issue(s)
[API-2793](https://vajira.max.gov/browse/API-2793)

### CODE REVIEW

**Comment by templeman15** [comment]
@ElleeB this looks good to me, I ran it locally as well. I'll let @edmangimelli review it though before we merge!

**Comment by templeman15** [comment]
@ElleeB your CI has failed. It looks like a security warning, I've experienced this in the past and had to pull master, then merge master into my branch and then push my branch up again.

**Review by LindseySaari — APPROVED** [review]

**Pull request merged**
